index.html

```diff
@@ -39,5 +39,143 @@ <h1>Hey there. I’m Mariane</br>
         </li>
     </ul>
 </main>
+<section class="action">
+    <a href="#"><img src="./images/Action.png" alt="Action"></a>
+</section>
+<section class="works-title">
+    <h1>My Recent Works</h1>
+    <img src="./images/Indicator.png" alt="Indicator">
+</section>
+<div class="works-section">
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="about-me">
+    <h1>About me</h1>
```

Comment:
Since you already have an h1 header tag in the headline section, kindly change this to h2 or h3 in conformity with standard HTML practices.

Reply:
Fixed